include/envoy/server/fatal_action_config.h

```diff
@@ -18,10 +19,9 @@ class FatalAction {
   virtual ~FatalAction() = default;
   /**
    * Callback function to run when we are crashing.
-   * @param current_object the object we were working on when we started
-   * crashing.
+   * @param objects a vector of objects we were working on when we started crashing.
```

Comment:
objects-> tracked_objects
vector -> span?

and  while you're in here, mind changing  we -> Envoy?